NitroxClient/GameLogic/Vehicles.cs

```diff
@@ -232,4 +234,11 @@ public static void RemoveNitroxEntitiesTagging(GameObject constructedObject)
             UnityEngine.Object.DestroyImmediate(nitroxEntity);
         }
     }
+
+    public static VehicleWorldEntity MakeVehicleEntity(GameObject constructedObject, NitroxId constructedObjectId, TechType techType, NitroxId constructorId = null)
```

Comment:
```suggestion
    public static VehicleWorldEntity BuildVehicleWorldEntity(GameObject constructedObject, NitroxId constructedObjectId, TechType techType, NitroxId constructorId = null)
```
We usually use Build prefix for method that acts like a """factory"""